.sequelizerc

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+ module.exports = {
+  "config": path.resolve('./server/config', 'config.json'),
+  "models-path": path.resolve('./server/models'),
+  "seeders-path": path.resolve('./server/seeders'),
+  "migrations-path": path.resolve('./migrations')
+};
```

Comment:
This looks really weird. The syntax you have in the file is Node-specific (which means it' should be JS file), you can't have modules in an `rc` dotted file. Please read more about this [here](http://docs.sequelizejs.com/manual/tutorial/migrations.html#the-sequelizerc-file). If you can't get it to work let me know and I can help you.

Reply:
It seems it is "correct", which is strange. sequelize must do some weird stuff with that file, but if you check, most of the `rc` files in other projects are JSON based. The config path was wrong so I updated that.

Currently, the setup of sequelize is very strange and it uses mixed ways. Will refactor it in the following days.

Reply:
Will pick up on this task #21 and reconfigure sequelize to use the proper config file every time.